ddp-workspace/projects/ddp-dsm-ui/src/app/shipping-search/shipping-search.component.ts

```diff
@@ -166,4 +182,8 @@ saveCompleted(): void{
     }
     return false;
   }
+
+  buttonDisabled(): boolean {
```

Comment:
**OPTIONAL**
You could use here getter instead:
**TS:**
get buttonDisabled() {...}
**HTML:**
<element [disabled]="buttonDisabled" />

Reply:
👍🏻 Good idea!